belgif-rest-problem/src/main/java/io/github/belgif/rest/problem/api/RetryAfterClientProblem.java

```diff
@@ -48,4 +49,25 @@ public void setRetryAfterSec(Long retryAfterSec) {
         this.retryAfterSec = retryAfterSec;
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        if (!super.equals(o)) {
```

Comment:
It's the default generated equals() and hashCode() from IntelliJ though, so I'm not sure if we should modify it as it will be inconsistent when we ever regenerate it, or when we generate it in other classes of the project.

If we really want to change this, we better modify the IntelliJ equals and hashCode templates at CBSS?